app/models/payment.rb

```diff
@@ -0,0 +1,14 @@
+class Payment < ApplicationRecord
+  belongs_to :claim
+  belongs_to :payroll_run
+
+  validates :award_amount, presence: true
+
+  %i[payroll_reference gross_value national_insurance employers_national_insurance].each do |attr|
```

Comment:
We missed `net_pay` from both these.

Reply:
No need to look over the attributes and call `validates` multiple times; `validates` accepts multiple attributes: 
```ruby
validates :payroll_reference, :gross_value, :national_insurance, :employers_national_insurance, presence: true, on: :upload
```

Reply:
I've fixed both these things.